### PR TITLE
Adds event attribute directive to package. Requires jquery

### PR DESCRIPTION
**Pull request by jweisbeck**

### CODE REVIEW

File: event/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+/**
+ * @ngdoc directive
+ * @name buildium.common.bdEvent
+ *
+ * @description
+ *
+ * Attach callbacks to arbitray DOM events.
+ *
+ * <!-- example -->
```
Comment (digitil) [inline event/index.js]: `@example` to be parsed by doc generator

there are other tags available for [directives](https://github.com/angular/angular.js/wiki/Writing-AngularJS-Documentation#angularjs-specific-ngdoc-directives) as well

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@buildium/angular-ui",
-  "version": "1.0.1",
+  "version": "1.0.2-0",
```
Comment (edvio4) [inline package.json]: i believe this should be 1.1.0-2 (i think ryan's pr is 1.1.0-1)?

Comment (jweisbeck) [reply package.json]: `npm version prerelease` bumped it to 1.0.2-0 ... @digitil which seems correct? I would've expected it to add a `-1`

**Review by ryan-pierce-buildium — APPROVED** [review]
Looks good

File: package.json (hunk 2 above)
Comment (digitil) [reply package.json]: you probably haven't updated from master @jweisbeck the prerelease should be on version 1.1.0

**Review by digitil — CHANGES_REQUESTED** [review]
update package version

**Review by digitil — APPROVED** [review]

**Pull request merged**